k2/k2/reports/junit/test/test_writer.py

```diff
@@ -4,6 +4,8 @@
 import unittest
 from textwrap import dedent
 
+from xmldiff import main
```

Comment:
Different python versions would output xml documents with attributes listed in different order, so I opted for using a 3rd party library to compare xml documents.